### PR TITLE
Creating the Botkube Flux Plugin for Day 2 operations

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Creating the Botkube Flux Plugin for Day 2 operations

Preview: https://blog-post.botkube-docs.pages.dev/blog

### CODE REVIEW

File: blog/executor-plugins.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    image_url: https://avatars.githubusercontent.com/u/17568639?s=250
+---
+
+# Creating the Botkube Flux Plugin for Day 2 operations
+
+Hey there, tech enthusiasts! Today we're diving into the world of GitOps with the brand-new Botkube Flux plugin.
+
+## Introduction
+
+In Botkube, we move towards GitOps by developing such extensions like Flux plugin. We all know the importance of GitOps workflow automation—it's the backbone of modern Kubernetes management. But sometimes, we encounter situations where automation is needed but hasn't been implemented yet. That's where Botkube steps in.
```
Comment (yllek) [inline blog/executor-plugins.md:16]: In Botkube, we move towards GitOps by developing extensions like the new [Flux plug-in](https://botkube.io/integration/botkube-flux-kubernetes-integration).

File: blog/executor-plugins.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+1. [Conditional requests](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#conditional-requests) because receiving a 304 response doesn't count against token rate limit.
+2. Adding support for GitHub App tokens. By using GitHub Apps, you can increase your maximum rate limits because multiple GitHub Apps are independent and do not share the rate limits. Where, using multiple Personal Access Tokens (PATs) for the same account will result in sharing the same rate limit.
+
+In the future, we can consider adding a token rotator that automatically switches tokens before hitting the rate limit.
+
+For the [Botkube web app](https://app.botkube.io/) we will consider native integration using GitHub App, to reduce friction with the initial setup for Flux and GitHub Events plugins.
+
+## Conclusion
+
+The Botkube Flux plugin offers valuable solutions for streamlining GitOps workflows. Its capabilities, including mobile-friendly interactivity and automated Day 2 operations support, can significantly enhance your Kubernetes management.
```
Comment (yllek) [inline blog/executor-plugins.md:180]: The [Botkube Flux plug-in](https://botkube.io/blog/introducing-botkubes-integration-with-flux)

File: blog/executor-plugins.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+We encourage you to explore the Botkube Flux plugin and consider integrating it into your workflows. Don't hesitate to share your feedback and ideas about Botkube. Feel free to reach out to us on Slack or Twitter.
```
Comment (yllek) [inline blog/executor-plugins.md:182]: Feel free to reach out to us on [Slack](https://join.botkube.io/) or [Twitter](http://twitter.com/botkube_io)

**Comment by mszostok** [comment]
Moved to Webflow with applied comments 👍  Thanks for the review!

**Pull request closed without merging**